ocs_ci/framework/conf/default_config.yaml

```diff
@@ -98,12 +98,12 @@ ENV_DATA:
   worker_replicas: 3
   skip_ocp_deployment: false
   skip_ocs_deployment: false
-  ocs_version: '4.3'
+  ocs_version: '4.4'
   # uncomment to use custom directory for storing measurement data related to
   # monitoring tests, otherwise generate temporary directory for each test run
   # measurement_dir: '/tmp/ocs_ci_monitoring_measurement/'
   # Default RHCOS image to be used for VmWare deployment
-  vm_template: 'rhcos-4.3.0-x86_64-vmware'
+  vm_template: 'rhcos-4.4.0-x86_64-vmware'
```

Comment:
didn't see above template in in downloads page. I see below as latest
rhcos-4.4.0-rc.1-x86_64-vmware.x86_64

Reply:
done